dictionary/README.md

```diff
@@ -3,3 +3,4 @@ Please write a couple of sentences explaining your choice of restart
 strategy.
 
 
+I chose to use transient here because the user may be able to have a normal or shutdown in which case the word dictionary should not be automatically restarted. The user may not want this dictionary running at all times and therefore should successfully stop when the user requests a normal termination.
```

Comment:
# -2

I see your point, but I'd prefer :permanent here because I think it better expresses intent—the dictionary should never stop while the overall application is running.